src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -109,6 +111,11 @@
 	private int remainingOperations;
 	private boolean somethingChanged;
 
+	public long lastTime;
+	public long elapsedTime;
+	public long startItemCount;
+	public long remainingItemCount;
+
```

Comment:
public for convenience currently. Mostly to fine tune the calculation.
Mutable is required as they are changing and the multiblock is not destroyed after each job is finished.
Basically first get it working, then make it pretty and use the PR to discuss.